packages/expo-router/src/typed-routes/generate.ts

```diff
@@ -34,6 +36,7 @@ export function getTypedRoutesDeclarationFile(
       ignoreEntryPoints: true,
       ignoreRequireErrors: true,
       importMode: 'async', // Don't load the file
+      unstable_useServerMiddleware,
```

Comment:
leaving a future note here to update `getRoutes` to not have optional options in the future (opportune for when we'll do the server/client split, most likely)